### PR TITLE
record: implement multiple communities inclusion

### PR DESCRIPTION
**Pull request by ntarocco**
* implements the new request type community-inclusion
* renames permission to align with the new request type
* closes inveniosoftware/invenio-app-rdm#2060

### CODE REVIEW

File: invenio_communities/permissions.py [hunk 1]
```diff
@@ -74,7 +74,8 @@ class CommunityPermissionPolicy(BasePermissionPolicy):
         ),
     ]
 
-    can_direct_publish = [
+    # who can include a record directly, without a review
+    can_include_directly = [
```
Comment (jrcastro2) [inline invenio_communities/permissions.py:78]: There are a couple of occurrences of this that should also be updated, see https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js#L60-L63

**Review by jrcastro2 — COMMENTED** [review]
Tests are failling due to some linting issues.

**Review by kpsherva — APPROVED** [review]

File: invenio_communities/permissions.py (hunk 1 above)
Comment (ntarocco) [reply invenio_communities/permissions.py:78]: Thank you, no idea how I missed those 👀

**Comment by ntarocco** [comment]
@kpsherva @jrcastro2 everything should be fixed now, thank you for the review.

**Pull request merged**